examples/labelanchor-example.ts

```diff
@@ -207,9 +233,34 @@ export class Position2DLabelExample extends Example {
         this._canvas.framePrecision = Wizard.Precision.byte;
         this._canvas.frameScale = [1.0, 1.0];
 
-        this._renderer = new Position2DLabelRenderer();
+        this._renderer = new LabelAnchorRenderer();
         this._canvas.renderer = this._renderer;
 
+        const hrStyle = 'z-index: 1; position: absolute; width: 99%; margin: 0;'
```

Comment:
Since this is an Example, please comment for what those HTML-lines are used.
This is also good for the future; this way we notice when the anchors are off.